src/test/java/gov/usgs/wma/waterdata/PruneTimeSeriesTest.java

```diff
@@ -0,0 +1,49 @@
+package gov.usgs.wma.waterdata;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import static gov.usgs.wma.waterdata.PruneTimeSeries.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(webEnvironment= SpringBootTest.WebEnvironment.NONE)
```

Comment:
In a pure Java unit test not really using the spring context like this, we can use the '@ExtendWith(SpringExtension.class)' annotation and not incur the overhead associated with the '@SpringBootTest'.

Reply:
Got it.  Done.